src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java

```diff
@@ -150,12 +153,27 @@ private void processJobUpdate(JobExecution jobExecution, boolean after) {
       }
       jobExecutionUpdate.setProgress(progress);
     }
+    populateFileNames(jobExecution, jobExecutionUpdate, jobParameters);
     kafka.send(KafkaService.Topic.JOB_UPDATE, jobExecutionUpdate.getId().toString(), jobExecutionUpdate);
     if (after) {
       log.info("-----------------------------JOB---ENDS-----------------------------");
     }
   }
 
+  private void populateFileNames(JobExecution jobExecution, Job jobExecutionUpdate, JobParameters jobParameters) {
```

Comment:
@obozhko-folio , that logic should be updated if you are not going to check type in scope of the mod-data-export-spring

Reply:
I returned back checking export type in mod-data-export-spring

Reply:
you shouldn't check type in mod-data-export-spring